.travis/custom_release.sh

```diff
@@ -4,6 +4,10 @@ NODE_ENV=production npm run build
 
 if [ "${TRAVIS_BRANCH}" = "master" ]; then
     .travis/release.sh "ci-beta"
-elif [[ "${TRAVIS_BRANCH}" = "ci-stable"  || "${TRAVIS_BRANCH}" = "qa-beta" || "${TRAVIS_BRANCH}" = "qa-stable" || "${TRAVIS_BRANCH}" = "prod-beta" || "${TRAVIS_BRANCH}" = "prod-stable" ]]; then
+    .travis/release.sh "qa-beta"
+elif [ "${TRAVIS_BRANCH}" = "master-stable" ]; then
```

Comment:
As discussed, maybe just "stable" is fine?

Reply:
Updated 👍 

@syncrou - Need to discuss this next week for Sources and Topo UIs.  Slight departure from other UI repos, but keeps better alignment with API which use master/stable branch names.